site2/docs/functions-deploy-cluster-builtin.md

```diff
@@ -0,0 +1,16 @@
+---
+id: functions-deploy-cluster-builtin
+title: Built-in functions
+sidebar_label: "Built-in functions"
```

Comment:
It's a task-oriented topic for users. So it makes more sense to use an imperative title, like `Use built-in functions`.

Reply:
done